core/src/main/java/hudson/security/Permission.java

```diff
@@ -315,7 +306,26 @@ public boolean getEnabled() {
      *      Access {@link jenkins.model.Jenkins#ADMINISTER} instead.
      */
     @Deprecated
-    public static final Permission HUDSON_ADMINISTER = new Permission(HUDSON_PERMISSIONS,"Administer", hudson.model.Messages._Hudson_AdministerPermission_Description(),null);
+    public static final Permission HUDSON_ADMINISTER;
+
+    @Restricted(NoExternalUse.class)
```

Comment:
Would you not want plugins to be checking this?

Reply:
I marked this restricted because the real instance is in `Jenkins.ROOT` to mirror `Jenkins.ADMINISTER`. The same initialization ordering constraints apply here, hence the permission is defined in `Permission`.